backtrace-library/src/androidTest/java/backtraceio/library/BacktraceClientBreadcrumbsTest.java

```diff
@@ -65,6 +64,8 @@ public void cleanUp() {
     public void sendBacktraceExceptionBreadcrumbs() {
         // GIVEN
         backtraceClient.enableBreadcrumbs(context);
+        // Account for mandatory configuration breadcrumb
+        backtraceClient.setCurrentBreadcrumbId(1);
```

Comment:
I'm really against exposing methods that can break stuff to public interface available to the user. Wouldn't it be better if:
* this method will be available only in the breadcrumbs class instance?
* in the unit test project we will fetch breadcrumbs class instance and cast it from interface to realy class instance?
* hide setCurrentBreadcrumbId in public interface (including backtraceclient and breadcrumb) and allow to do that only from class implementation?

Reply:
Will do, thank you!